shared/common-adapters/button.js

```diff
@@ -85,16 +86,20 @@ class Button extends Component<Props> {
         <Box
           style={{
             ...globalStyles.flexBoxCenter,
+            ...globalStyles.flexBoxRow,
             position: 'relative',
             height: '100%',
           }}
         >
-          <Text
-            type={this.props.small ? 'BodySemibold' : 'BodyBig'}
-            style={{...labelStyle, ...this.props.labelStyle}}
-          >
-            {this.props.label}
-          </Text>
+          {this.props.children}
```

Comment:
can't we just have both at the same time, just have children and text as siblings